public/js/lib/devtools-sham/shared/event-emitter.js

```diff
@@ -117,15 +117,15 @@ EventEmitter.prototype = {
       // event handler we're going to fire wasn't removed.
       if (originalListeners === this._eventEmitterListeners.get(aEvent) ||
           this._eventEmitterListeners.get(aEvent).some(l => l === listener)) {
-        try {
+        // try {
           listener.apply(null, arguments);
-        }
-        catch (ex) {
-          // Prevent a bad listener from interfering with the others.
-          let msg = ex + ": " + ex.stack;
-          //console.error(msg);
-          console.log(msg);
-        }
+        // }
+        // catch (ex) {
+        //   // Prevent a bad listener from interfering with the others.
+        //   let msg = ex + ": " + ex.stack;
+        //   //console.error(msg);
+        //   console.log(msg);
+        // }
```

Comment:
I'm not sure, it was something from the client stuff. I'm not sure what the implications are; I get the desire to make sure that all event listeners are fired, and if one errors it doesn't stop the whole process. Eventually we will need to share this code exactly as it exists on m-c, so we could have a discussion for changing it there. Until then I think I'll just revert this.